client/post-editor/editor-post-formats/accordion.jsx

```diff
@@ -1,68 +1,64 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	pick = require( 'lodash/pick' ),
-	find = require( 'lodash/find' ),
-	classNames = require( 'classnames' ),
-	isEmpty = require( 'lodash/isEmpty' );
+import React, { PropTypes } from 'react';
+import { has, isEmpty } from 'lodash';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var Accordion = require( 'components/accordion' ),
-	Gridicon = require( 'components/gridicon' ),
-	siteUtils = require( 'lib/site/utils' ),
-	PostFormats = require( './' );
+import Accordion from 'components/accordion';
+import Gridicon from 'components/gridicon';
+import siteUtils from 'lib/site/utils';
+import PostFormats from './';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 	displayName: 'EditorPostFormatsAccordion',
 
 	propTypes: {
-		site: React.PropTypes.object,
-		post: React.PropTypes.object,
-		postFormats: React.PropTypes.arrayOf( React.PropTypes.shape( {
-			slug: React.PropTypes.string,
-			label: React.PropTypes.string
-		} ) )
```

Comment:
I respect your point of view and It's a good compromise for me If you want to keep this part as is. but I can explain more my point of view.

I think storing it as an object in Redux is great, because we can write performant selectors based on this. We can write `getPostFormats` selector which we could call if we want to have the full list (array) while we can write another selector `getPostFormat(state, slug)` to access directly the required post format by Slug. Maybe I would also add another selector called `getPostFormatsBySlug` which returns the object by slug.